draft-ietf-oauth-identity-chaining.md

```diff
@@ -164,6 +170,12 @@ All of section 2.2 of {{RFC8693}} applies. In addition, the following applies to
 
 * The "aud" claim included in the returned JWT authorization grant MAY identify multiple authorization servers, provided that trust relationships exist with them (e.g. through federation). It is RECOMMENDED that the "aud" claim is restricted to a single authorization server to prevent an authorization server in one domain from presenting the client's authorization grant to an authorization server in a different trust domain. For example, this will prevent the authorization server in Domain B from presenting the authorization grant it received from the client in Domain A to the authorization server for Domain C.
 
+* If the token exchange request in (B) contains a proof of possession of a private key, the following apply.
+  * AS-A MUST verify the proof of possession for the token in the request (the initial inbound token)
```

Comment:
I'm not sure how AS-A would be able to verify the proof of possession:
- If mTLS is used the connecting peer of AS-A is RS-A, which key would differ to the one present in the cnf claim ("clients" key in "cnf" vs "RS" key in mTLS)
- If DPoP the proof could be "forwarded" but how would AS-A know what audience, htm, htu to expect? And they'll differ to the endpoint and method used during token exchange.

Reply:
I only realized during the meeting that the `cnf` key of the resulting access token will not be the same as the `cnf` key of the incoming access tokens.
Maybe it would be good to mention this in the draft. E.g. 

```
* If the JWT authorization grant in the access token request contains a "requested_cnf" claim **with the key of the client performing the token exchange. This will bind the access token for domain B to the client planning to use it.**
```